shared/src/persistence/elasticsearch/fetchPendingItems.js

```diff
@@ -1,19 +1,24 @@
+const { isCodeEnabled } = require('../../../../codeToggles');
 const { search } = require('./searchClient');
 
-exports.fetchPendingItems = async ({ applicationContext, judge, page }) => {
-  const caseSource = [
-    'associatedJudge',
-    'caseCaption',
-    'docketNumber',
-    'docketNumberSuffix',
-    'status',
-  ];
-  const docketEntrySource = [
-    'docketEntryId',
-    'documentType',
-    'documentTitle',
-    'receivedAt',
-  ];
+exports.fetchPendingItems = async ({
+  applicationContext,
+  judge,
+  page,
+  source,
+}) => {
+  const caseSource = isCodeEnabled(7134)
+    ? [
+        'associatedJudge',
+        'caseCaption',
+        'docketNumber',
+        'docketNumberSuffix',
+        'status',
+      ]
+    : source;
+  const docketEntrySource = isCodeEnabled(7134)
+    ? ['docketEntryId', 'documentType', 'documentTitle', 'receivedAt']
```

Comment:
Same as above - when set to 'true', this toggle appears to be keeping the same functionality as was there previously. Is that the intention?

Reply:
This was also a part of the double toggle we had to implement!